### ISSUE TITLE
Integrate prettier for `--fix`

### ISSUE THREAD

**Issue by BrandonOCasey** [videojs/standard#37]
This should allow us to have much more control over the resulting code from `--fix` 

See: https://github.com/prettier/prettier

**Comment by gkatsev** [comment]
I don't think we should do this. We should either use prettier and integrate it as a precommit hook or something or just continue using eslint.
I'm leaning towards just staying with eslint for now, especially since --fix works with quite a lot of stuff now.
